velox/dwio/dwrf/test/ReaderTest.cpp

```diff
@@ -1942,12 +1941,10 @@ void testFlatmapAsMapFieldLifeCycle(
   EXPECT_EQ(keysPtr, child->mapKeys().get());
   // there is a TODO in FlatMapColumnReader next() (result is not reused)
   // should be EQ; fix: https://fburl.com/code/wtrq8r5q
-  // EXPECT_EQ(childPtr, child.get());
```

Comment:
Better leave this there although I don't think anyone is going to work on these TODOs in short term